lib/python/treadmill_aws/cli/admin/aws/image.py

```diff
@@ -0,0 +1,50 @@
+"""Implementation of treadmill admin EC2 image.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import click
+
+from treadmill import cli
+
+from treadmill_aws import awscontext
+from treadmill_aws import ec2client
+
+
+def init():
+
+    """EC2 image CLI group"""
+    formatter = cli.make_formatter('aws_image')
+
+    @click.group()
+    def image():
+        """Manage image configuration"""
+        pass
+
+    @image.command(name='list')
+    @cli.admin.ON_EXCEPTIONS
+    def _list():
+        """List images"""
+        ec2_conn = awscontext.GLOBAL.ec2
+        _account = awscontext.GLOBAL.session.client(
```

Comment:
it is ok for now, but i suggest you follow the convention and add .sts property to aws context similar to ec2/s3 etc. it is not ideal, and i would like to refactor it to some dynamic getattr thing, but it will be easier to refactor if we keep all of them in one place.